### DISCUSSION TITLE
Secret "microcks-microcks-grpc-secret" not found

### DISCUSSION THREAD

**Discussion by CROprogrammer** [microcks/microcks#703] [Q&A] [answered]
When I'm trying to install Microcks using Helm and Minikube then I'm getting error "MountVolume.SetUp failed for volume "microcks-grpc-certs" : secret "microcks-microcks-grpc-secret" not found" that can be found on Minikube dashboard

![image](https://user-images.githubusercontent.com/73464772/204013222-2fd2237c-983b-42b0-a332-b46481eb1e77.png)

**Comment by lbroudoux** [comment]
Is the secret created? Can you list the secrets within the namespace you installed Microcks? `kubectl get secrets` result?

**Reply by CROprogrammer** [reply]
Secret isn't created. How do I create one?

![image](https://user-images.githubusercontent.com/73464772/204019055-4edcbc81-da72-4eae-af76-e3acb98efd01.png)

**Reply by lbroudoux** [reply]
Mmm ... It should be created. Let me try to reproduce it...

**Reply by lbroudoux** [reply]
Hey you're right! I made a mistake whilst releasing 1.6.1... I'm gonna fix it right now...

**Comment by lbroudoux** [comment, marked as answer]
I've redeployed the `1.6.1` version of the Helm chart onto our repo. Just refresh/overwrite your local version and secret should now be created. 
Thanks a lot for your feedback!

**Reply by CROprogrammer** [reply]
Do you know why is it still an issue when I'm trying to install from the sources cloned locally?

**Reply by lbroudoux** [reply]
Fix was pushed on branch `1.6.x`and not on `master`

**Reply by CROprogrammer** [reply]
I'm still having trouble with the same issue even after using branch 1.6.x instead of master. Any idea why?

**Reply by lbroudoux** [reply]
My bad, sorry, I forgot to push on GitHub. It should be ok after a refresh.
